gmmp/dashboard_modules.py

```diff
@@ -96,31 +96,34 @@ class Submissions(DashboardModule):
     title = "Coded"
     template = "gmmp/dashboard_modules/submissions.html"
 
+    def get_count(self, model_, user):
```

Comment:
Isn't this for "internal" use only & hence should start with `_` i.e. `_get_count`?